src/main/java/uk/nhs/digital/nhsconnect/nhais/service/OutboundQueueService.java

```diff
@@ -48,13 +48,13 @@ public void receive(Message message) throws IOException, JMSException {
         try {
             String body = JmsReader.readMessage(message);
             LOGGER.debug("Received message body: {}", body);
-            OutboundMeshMessage meshMessage = objectMapper.readValue(body, OutboundMeshMessage.class);
-            LOGGER.debug("Decoded message: {}", meshMessage);
-            String recipient = meshCypherDecoder.getRecipient(meshMessage);
+            OutboundMeshMessage outboundMeshMessage = objectMapper.readValue(body, OutboundMeshMessage.class);
+            LOGGER.debug("Decoded message: {}", outboundMeshMessage);
+            String recipient = meshCypherDecoder.getRecipient(outboundMeshMessage);
             meshClient.sendEdifactMessage(
```

Comment:
Shouldn't we pass `OutboundMeshMessage` here and decode recipient inside `MeshClient`? Decoding recipient into mesh mailbox id seems more related to `MeshClient` than to Inbound queue